sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -551,9 +551,6 @@ def test_reshuffle_preserves_timestamps(self):
               {
                   'name': 'bar', 'timestamp': 33
               },
```

Comment:
Can we un-sickbay this test after this fix?

Reply:
I'm afraid not.
It seems b/146457921 and BEAM-9003 are two different issues.
b/146457921 is for Unified Worker and BEAM-9003 is for Dataflow.

Reply:
I'm not a member, I can't run the Jenkins VR test or see the results, so I can't say much about  BEAM-9003.

Reply:
Valentyn helped reproduce BEAM-9003 bug on Dataflow. The pipeline runs forever (timeout) on Dataflow streaming. So maybe BEAM-9003 is caused by a different reason.

We'll concentrate on b/146457921 here.

Reply:
I'll try to run the test on VR to see if it passes.